git-switch.rb

```diff
@@ -34,7 +33,12 @@
 when :modified
   branches = `git for-each-ref --format="%(refname:short)" --sort='-authordate' refs/heads --count #{options[:count]}`.split
 else
-  branches = `git reflog | grep "checkout: moving" | cut -d' ' -f8`.split.uniq.drop(1).take(options[:count])
+  branches = `git reflog | grep "checkout: moving" | cut -d' ' -f8`.split.uniq.drop(1)
+
+  # Remove deleted branches from the result set
+  branches &= `git branch`.split unless options[:show_deleted_branches]
```

Comment:
The option `:show_deleted_branches` is never declared.

In my opinion this option shouldn't exist since with a true value git-switch creates a new branch, which doesn't seem to be the intention of the user.
What do you think?